docs/blog/articles/2024-05-15-Kserve-0.13-release.md

```diff
@@ -0,0 +1,75 @@
+
+Here is an example Inference Service with HuggingFace runtime. See more details in our updated docs to [Deploy the Llama2 model with Hugging Face LLM Serving Runtime](https://kserve.github.io/website/master/modelserving/v1beta1/llm/huggingface/) and our python SDK [HuggingFace Server Runtime docs](https://github.com/kserve/kserve/tree/master/python/huggingfaceserver).
+
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: huggingface-llama2
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: huggingface
+      args:
+      - --model_name=llama2
+      - --model_id=meta-llama/Llama-2-7b-chat-hf
+      resources:
+        limits:
+          cpu: "6"
+          memory: 24Gi
+          nvidia.com/gpu: "1"
+        requests:
+          cpu: "6"
+          memory: 24Gi
+          nvidia.com/gpu: "1"
+```
+
+### vLLM support 
+
+Version 0.13 introduces dedicated runtime support for [vLLM](https://docs.vllm.ai/en/latest/), for enhanced transformer model serving. This support now includes auto-mapping vLLMs as the backend for supported tasks, streamlining the deployment process and optimizing performance.
+
+
+### OpenAI Schema Integration
+
+Embracing the OpenAI protocol, KServe v0.13 now supports three specific endpoints:
+
+* `/v1/completions`
+* `/v1/chat/completions`
+* `/v1/models`
+
+This update fosters a standardized approach to transformer model serving, ensuring compatibility with a broader spectrum of models and tools, and enhances the platform's versatility.
+
+
+## Key Changes
+
+This release includes several fixes key issues and added features:
+
+* Async streaming support for v1 endpoints. (#3402)
+* Improved scaling behaviors under high concurrency scenarios (#3487).
+* Support for .json and .ubj model formats in the XGBoost server image (#3546).
+* Enhanced flexibility in KServe by allowing the configuration of multiple domains for an inference service (#2747).
+* Enhanced the manager setup to dynamically adapt based on available CRDs, improving operational flexibility and reliability across different deployment environments (#3470).
+* Removal of conversion webhook from manifests (#3344)
+
+## Join the community
+
+- Visit our [Website](https://kserve.github.io/website/) or [GitHub](https://github.com/kserve)
+- Join the Slack ([#kserve](https://kubeflow.slack.com/?redir=%2Farchives%2FCH6E58LNP))
+- Attend our community meeting by subscribing to the [KServe calendar](https://wiki.lfaidata.foundation/display/kserve/calendars).
+- View our [community github repository](https://github.com/kserve/community) to learn how to make contributions. We are excited to work with you to make KServe better and promote its adoption!
+
+For complete details on the new features and updates, visit our [official release notes](https://github.com/kserve/kserve/releases/tag/v0.13.0-rc0).
+
+Thanks for all the contributors who have made the commits to 0.13 release!
+
+The KServe Working Group
```

Comment:
KServe project

Reply:
Also nice to acknowledge the major contributors for this release :)

Reply:
what do you consider "major"?

Reply:
"project" would make more sense to me. We can also remove this line :-)